### DISCUSSION TITLE
Key binding does not function consistently when menus are open in various programs

### DISCUSSION THREAD

**Discussion by RayCulp** [flameshot-org/flameshot#3255] [Q&A] [answered]
Hi everyone,

I'm using Ubuntu 22.04.2 and Flameshot 11.0.0-2 Debian. I set up a global key binding for `flameshot gui` and it works well in most cases, but I've noticed that it does unexpected things sometimes.

When creating a tutorial for a program, it is useful to be able to display an open menu to show the user which menu item to click. For example, if I try to take a screenshot of an open menu in the file explorer, it works as expected:

![image](https://github.com/flameshot-org/flameshot/assets/7621330/6d1ca0f2-d619-4668-bb52-054666cc8544)

However, if I try to take a screenshot in Firefox with a menu open, the menu gets closed before the screenshot is taken. If I try it in Audacity, Flameshot does not respond at all.

Has anyone else noticed this ?

**Comment by mmahmoudian** [comment, marked as answer]
This is intended behavior of almost all window managers and has nothing to do with Flameshot. Your window manager or desktop environment (in this case Gnome) is not letting the keybinding to pass. You can try creating a shortcut for some program other than Flameshot and see that also does not get triggered.


If you want to capture a menu, the best way is to run Flameshot with a delay:

```sh
flameshot gui --delay 5000
```

or

![image](https://github.com/flameshot-org/flameshot/assets/390889/8d3d4b64-c7ef-406e-86cb-c5e2ff91b928)

**Reply by RayCulp** [reply]
@mmahmoudian Thank you, this works perfectly.
